0001-kaa-protocol/README.md

```diff
@@ -1,9 +1,9 @@
 ---
-name: Kaa Protocol
+name: Kaa protocol
 shortname: 1/KP
 status: draft
-editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
-contributors: Alexey Gamov <agamov@cybervisiontech.com>, Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
+editor: Alexey Shmalko<ashmalko@cybervisiontech.com>
+contributors: Alexey Gamov<agamov@cybervisiontech.com>, Andrew Kokhanovskyi<akokhanovskyi@cybervisiontech.com>
```

Comment:
That's GitHub rendering issue.

The reason to leave space is that many software uses that format (`User Name <user@email>`) (from the top of my head: git, gpg, gnus, most mail clients).